ubuntu-advantage

```diff
@@ -15,7 +15,7 @@ FIPS_REPO_URL="private-ppa.launchpad.net/ubuntu-advantage/fips"
 FIPS_REPO_KEY_FILE="ubuntu-fips-keyring.gpg"
 FIPS_REPO_LIST=${FIPS_REPO_LIST:-"/etc/apt/sources.list.d/ubuntu-fips-${SERIES}.list"}
 REPO_URL="esm.ubuntu.com"
-REPO_KEY_FILES="ubuntu-esm-keyring.gpg ubuntu-trusty-esm-keyring.gpg"
+REPO_KEY_FILES="ubuntu-esm-keyring.gpg ubuntu-esm-v2-keyring.gpg"
```

Comment:
You need to remove the original ESM key file from this list. You deleted it from the package so later on it cannot be used and this variable is used to cp files into apt's keyring.

Reply:
I did not delete it from the package. I deleted an extra copy that was in the root directory of the repository. The one we are installing is in the keyrings/ directory.